work_dir_tools.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+#variables
+MODE=$1
+SDL_BACKUP=${TMPDIR-/tmp}"/sdL_backup"
+SDL_TMP=${TMPDIR-/tmp}"/fs/mp/images/ivsu_cache"
+SDL=${2%/}
+
+#functions
+usage() {
+    echo "Usage: env.sh [OPTION] [PATH]"
+    echo "Path is a link to SDL bin folder"
+    echo "Mode can be the following:"
+    echo "  -b, --backup - Back up SDL configuration"
+    echo "  -c, --clean - Remove temporary SDL working files"
+    echo "  -r, --restore - Restore SDL configuration from backup"
+    echo "  -h, --help - Display this help"
+}
+
+backup() {
+	echo "Backup environment"
+	rm -r -f $SDL_BACKUP
+	mkdir $SDL_BACKUP
+	if [ -d $SDL_BACKUP ]; then
+		cp $SDL"/hmi_capabilities.json" $SDL_BACKUP
```

Comment:
@LuxoftAKutsan 
Error like "cp: cannot stat ‘hmi_capabilities.json’: No such file or directory" will be displayed